.travis.yml

```diff
@@ -2,11 +2,16 @@ language: php
 
 sudo: false
 
-php:
-    - 5.5
-    - 5.6
-    - 7.0
-    - hhvm
+matrix:
```

Comment:
By default we are running the precise dist on travis, which has the best support for PHP. HHVM support on precise is lagging behind at version 3.6.6, however. Making sure we run the HHVM build against the trusty dist will provide us with the latest version available.

https://docs.travis-ci.com/user/languages/php#HHVM-versions-on-Trusty